include/PathUtil.h

```diff
@@ -0,0 +1,27 @@
+#ifndef _PATHUTIL
+#define _PATHUTIL
+
+#include <QDir>
+
+namespace PathUtil
+{
+	enum Base { AbsoluteBase, FactoryBase, SampleBase, VSTBase, SoundfontBase };
+
+	QString baseLocation( Base base );
```

Comment:
You should include `lmms_export.h` and add `LMMS_EXPORT` before the function name(only in the header file).